fixes broken audio at WP1 and WC
Fill audio packets completely when resampling to prevent 'audio data unaligned' kernel warnings

--- a/xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEBuffer.cpp
+++ b/xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEBuffer.cpp
@@ -143,6 +143,7 @@
     m_inputFormat.m_channelLayout.Reset();
     m_inputFormat.m_channelLayout += AE_CH_FC;
   }
+  m_fillPackets = true;
   m_resampleQuality = quality;
 }
 
